ci-operator/step-registry/openshift/microshift/infra/wait-for-cluster-up-aws/openshift-microshift-infra-wait-for-cluster-up-aws-commands.sh

```diff
@@ -56,9 +56,28 @@ ssh "${INSTANCE_PREFIX}" "/home/${HOST_USER}/start_microshift.sh"
 
 ssh "${INSTANCE_PREFIX}" "sudo cat /var/lib/microshift/resources/kubeadmin/${IP_ADDRESS}/kubeconfig" >/tmp/kubeconfig
 
-KUBECONFIG=/tmp/kubeconfig oc wait \
-  pod \
-  --for=condition=ready \
-  -l='app.kubernetes.io/name=topolvm-csi-driver' \
-  -n openshift-storage \
-  --timeout=10m
+retries=3
+while [ ${retries} -gt 0 ] ; do
+  ((retries-=1))
+
+  set +e
+  KUBECONFIG=/tmp/kubeconfig oc wait \
```

Comment:
could we log how long we waited? we could use it to tweak the timeout
maybe `set -x` would be useful because rn it doesn't log what command runs

Reply:
but we can do it in a separate PR if you want

Reply:
Done